cellengine/resources/compensation.py

```diff
@@ -27,6 +26,27 @@ def create(cls, experiment_id: str, compensation: dict) -> Compensation:
         """
         return ce.APIClient().post_compensation(experiment_id, compensation)
 
+    @staticmethod
+    def from_spill_string(spill_string: str) -> Compensation:
+        """Creates a Compensation from a spill string (a file-internal compensation).
+        This can be used with FcsFile.spill_string. The compensation is not
+        saved to CellEngine.
+        """
+        arr = (
+            spill_string.replace("\n", "").replace("'", "").replace(" ", "").split(",")
```

Comment:
I think this should just be `spill_string.split(",")`. No newlines are allowed in it, and `'` and ` ` should be respected, unless something is modifying the channel names in the Python FCS parser. (Something in the R toolkit/R is modifying channel names, extremely annoying behavior there.)

Looks like this was motivated by the linebreaks in the test.